From: Sven Eckelmann <sven@narfation.org>
Date: Mon, 23 Jan 2017 18:59:17 +0100
Subject: batman-adv: Change maintainer to Simon Wunderlich

Marek Lindner forwarded all recent requests regarding the package changes
to Simon. He also asked that further changes should be handled directly
without waiting for him.

Simon Wunderlich already has write access to the repository and is
therefore well prepared to take over the role of the maintainer.

Signed-off-by: Sven Eckelmann <sven@narfation.org>

Origin: upstream, https://github.com/openwrt-routing/packages/commit/5e6eede6f32e14a8536902defbb8a846c640f7cf

diff --git a/batman-adv/Makefile b/batman-adv/Makefile
index 4c8e74082475e32f36a84c4172746b23e2efad89..10314fa023058921caf4b6c8841886bbaeac24c4 100644
--- a/batman-adv/Makefile
+++ b/batman-adv/Makefile
@@ -23,7 +23,7 @@ include $(INCLUDE_DIR)/package.mk
 
 define KernelPackage/batman-adv
   URL:=https://www.open-mesh.org/
-  MAINTAINER:=Marek Lindner <mareklindner@neomailbox.ch>
+  MAINTAINER:=Simon Wunderlich <sw@simonwunderlich.de>
   SUBMENU:=Network Support
   DEPENDS:=+KMOD_BATMAN_ADV_BLA:kmod-lib-crc16 +kmod-crypto-crc32c +kmod-lib-crc32c +kmod-cfg80211
   TITLE:=B.A.T.M.A.N. Adv
